Packs/CommonWidgets/ReleaseNotes/1_0_7.md

```diff
@@ -0,0 +1,7 @@
+
+#### Widgets
+##### New: MTTR by Type(in minutes)
+- Shows changes in Mean Time to Resolution(in minutes), over time, while differentiating between incident types.
```

Comment:
```suggestion
- Shows changes in Mean Time to Resolution (in minutes), over time, while differentiating between incident types.
```
In minutes or hours? The description says hours.

Reply:
minutes. i'll change the desc as well.